clinica/pydra/t1_volume/utils.py

```diff
@@ -0,0 +1,62 @@
+from os import PathLike
+from typing import Tuple, Union
+
+
+def initialize_tissues_spm_segment(
+    parameters: dict, tissue_probability_map: Union[PathLike, None] = None
+) -> tuple:
+    """Prepare data structure for SPM segment interface
+
+    Parameters
+    ----------
+    tissue_probability_map: PathLike, optional
+        Path to the nifti tissue probability map. Default=None.
+
+    Returns
+    -------
+    tuple
+        "tissues" data structure for SPMSegment
+    """
+
+    from clinica.pipelines.t1_volume_tissue_segmentation.t1_volume_tissue_segmentation_utils import (
+        get_tissue_tuples,
+    )
+    from clinica.utils.spm import get_tpm
+
+    parameters["tissue_probability_maps"] = tissue_probability_map or get_tpm()
+
+    tissue_tuples = get_tissue_tuples(
+        parameters["tissue_probability_maps"],
+        parameters["tissue_classes"],
+        parameters["dartel_tissues"],
+        parameters["save_warped_unmodulated"],
+        parameters["save_warped_modulated"],
+    )
+    return tissue_tuples
+
+
+def init_input_node(bids_name: str) -> Tuple[str, str]:
```

Comment:
I might be missing something, but I don't see where you use this function?

Reply:
You are right, the purpose of the node in the original pipeline was to print informative messages about the subjects to be executed. 

I suppose if we want to add something like this in Pydra we ought to pass by the engine input node instead of doing it pipeline by pipeline. Removed.